pkg/moov/institutions_models.go

```diff
@@ -41,3 +42,17 @@ type WireServices struct {
 	// The institution's capability to handle transfers of securities.
 	BookEntrySecuritiesTransferStatus bool `json:"bookEntrySecuritiesTransferStatus"`
 }
+
+type FedNowInstitution struct {
+	Name          string         `json:"name"`
+	RoutingNumber string         `json:"routingNumber"`
+	Address       *Address       `json:"address,omitempty"`
```

Comment:
There is no address in the compiled models

https://github.com/moovfinancial/moov-api-models/blob/main/go/v20250700/model_fed_now_institution.go#L22